src/main/startSandbox.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+import unpackByOutpoint from './unpackByOutpoint';
+
+// Polyfills and `lbry-redux`
+global.fetch = require('node-fetch');
+
+global.window = global;
+// eslint-disable-next-line import/no-commonjs,global-require
+const { Lbry } = require('lbry-redux');
+
+delete global.window;
```

Comment:
Why delete this?

Reply:
Should never try to emulate window inside Node, can cause side effects with other applications.
This is a patch to force this to work with `lbry-redux`, which I don't need, but `Lbry` is bundled with it, which I do need so I can make a daemon call for the file list.

See line 7.  This was grouped (spacing) with line 7.  The linter forced my grouping to break.